Framework/Core/src/DataSamplingConditionRandom.cxx

```diff
@@ -0,0 +1,71 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file DataSamplingConditionRandom.cxx
+/// \brief Implementation of random DataSamplingCondition
+///
+/// \author Piotr Konopka, piotr.jan.konopka@cern.ch
+
+#include "Framework/DataSamplingCondition.h"
+#include "Framework/DataProcessingHeader.h"
+
+#include <TRandom3.h>
```

Comment:
I usually use TRandom3 as well, but just for your information there are many other such classes in ROOT : https://root.cern.ch/doc/v612/group__Random.html